test/e2e/bm.e2e.test.ts

```diff
@@ -9,78 +10,85 @@ import { zimdumpAvailable, zimdump } from '../util.js'
 jest.setTimeout(200000)
 
 describe('bm', () => {
-  const now = new Date()
-  const testId = `mwo-test-${+now}`
-
-  const parameters = {
-    mwUrl: 'https://bm.wikipedia.org',
-    adminEmail: 'test@kiwix.org',
-    outputDirectory: testId,
-    redis: process.env.REDIS,
-    format: ['nopic'],
-  }
-
-  test('Simple articleList', async () => {
-    await execa('redis-cli flushall', { shell: true })
-
-    const outFiles = await mwoffliner.execute(parameters)
-
-    // Created 1 output
-    expect(outFiles).toHaveLength(1)
-
-    for (const dump of outFiles) {
-      if (dump.nopic) {
-        // nopic has enough files
-        expect(dump.status.files.success).toBeGreaterThan(14)
-        // nopic has enough redirects
-        expect(dump.status.redirects.written).toBeGreaterThan(170)
-        // nopic has enough articles
-        expect(dump.status.articles.success).toBeGreaterThan(700)
+  for (const renderer of renderers) {
+    // Test only render API that is supported by the wiki
```

Comment:
why, we want to test all end-points?! we should better want to test against a wiki which support all end-points... which is the case here anyway.

Reply:
so why we do have `if (renderer === 'WikimediaDesktop)'`, the loop should apply for all renderers, not only `WikimediaDesktop`.

Reply:
Because `bm.wikipedia.org` (which is the target of this test) doesn't support VE. When it meets VisualEdtitor it will fail, so I explicitly set `if (renderer === 'WikimediaDesktop)'. If it was `en.wikipedia.org`, your concern would be valid. But here we are faced with a problem about many-to-may relationships in tests: check multiple wikis with multiple endpoints at one go.

Reply:
VE does work on bm.wikipedia.org, check help on api.php endpoint.

Reply:
Yes, you're right, I've updated bm.e2e.test.ts